<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>224</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>工程目录：（下面用不到的选项可不填写）</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLineEdit" name="edProjectDir"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnSelectProjectDir">
       <property name="text">
        <string>选择</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>工程类型：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbProjectType"/>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>工程名称：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edProjectName"/>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>编译器：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbCompileType"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>编译器路径（msvc不用填）：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edCompilePath"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnCompile">
       <property name="text">
        <string>选择</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>字体：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFontComboBox" name="fontComboBox"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>字体大小：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbFontSize"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Boost目录：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edBoostPath"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnSelectBoost">
       <property name="text">
        <string>选择</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Qt目录：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edQtPath"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnSelectQt">
       <property name="text">
        <string>选择</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="text">
        <string>保存设置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnGenereate">
       <property name="text">
        <string>生成工程</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnExit">
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
